app/views/govuk_publishing_components/components/_select.html.erb

```diff
@@ -10,7 +10,7 @@
     <label class="govuk-label" for="<%= id %>">
       <%= label %>
     </label>
-    <select class="govuk-select" id="<%= id %>" name="<%= id %>">
+    <select class="govuk-select" id="<%= id %>" name="<%= id %>" data-module="<%= "track-select-change" if select_helper.data_tracking? %>" >
```

Comment:
Very small quibble here - this adds an empty `data-module` attribute to the element if data_tracking isn't true. Would be cleaner if it didn't.

Reply:
@andysellick Done - ready for re-review 🙂